### ISSUE TITLE
[Tarea 2] Pregunta 2 - Operaciones básicas

### ISSUE THREAD

**Issue by VicenteMerino** [UC-IIC3253/2021#33]
Hola, en la pregunta 2, dice que solamente podemos usar operaciones como suma, resta, división, multiplicación y resto. Pero por ejemplo para el alg. de euclides necesitamos también la operación "piso", está permitida usarla?

**Comment by marceloarenassaavedra** [comment]
Sí, se puede utilizar la operación piso (que corresponde a la división entera).

Saludos!

Marcelo

**Comment by Arcoirisky** [comment]
Añadiendome a la pregunta, podemos usar la operación & para ver si un número es par o impar?
